conf/machine/arp.conf

```diff
@@ -0,0 +1,8 @@
+#@TYPE: Machine
+#@NAME: arp
+
+#@DESCRIPTION: Machine configuration for Automotive Reference Platform
+
+require conf/machine/intel-corei7-64.conf
```

Comment:
Do we have this conf file?

Reply:
Yes this is part of meta-intel layer

Reply:
Alright.

Reply:
Is the ARP only usable with intel-corei7-64? Does it make sense to abstract/split this in some way?

Reply:
As far as I understood, ARP will only support intel cpu. May be its too premature to abstract it at the moment when we don't know if there will be other CPUs for ARP.

Reply:
Alright. Sounds good to me then.